packages/gamut-labs/src/ScoreSummary/index.tsx

```diff
@@ -63,8 +66,26 @@ const renderSubScores = ({
           px={16}
           py={8}
           justifyContent="space-between"
+          alignItems="center"
         >
-          <Text fontWeight="bold">{subContentTitle}</Text>
+          <FlexBox alignItems="center">
+            {colorfulIcons ? (
+              subContentPercentCorrect > 0.7 ? (
```

Comment:
```suggestion
              subContentPercentCorrect >= 0.7 ? (
```
Less than or equal to 70% right?